### PR TITLE
Update deprecated and new MITRE + Unit 42 display names

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2059)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Displays the available feeds from the MITRE taxii service.
 
 ##### Base Command
 
-`mitre-showfeeds`
+`mitre-show-feeds`
```
Comment (Ni-Knight) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md:100]: Is this just a readme fix?

Comment (dansterenson) [reply Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md:100]: yes, the command in the .yml file is `mitre-show-feeds` and in the README it was mitre-showfeeds by mistake I guess. The validation raised an error regarding the name differences.

**Review by Ni-Knight — APPROVED** [review]

File: Packs/FeedMitreAttackv2/TestPlaybooks/FeedMitreAttackv2_test.yml [hunk 2]
```diff
@@ -247,7 +247,7 @@ tasks:
             iscontext: true
           right:
             value:
-              simple: Two-Factor Authentication Interception
+              simple: Multi-Factor Authentication Interception
```
Comment (dansterenson) [inline Packs/FeedMitreAttackv2/TestPlaybooks/FeedMitreAttackv2_test.yml:250]: The APi value was changed and caused the test-playbook to fail.

**Comment by yaakovi** [comment]
Not updating the docker image cause it raises a run-time warning in the integrations.

**Pull request merged**